_data/collection-index.yml

```
@@ -393,3 +393,8 @@
   contact: https://github.com/joedmck/devcontainer-features/issues
   repository: https://github.com/joedmck/devcontainer-features
   ociReference: ghcr.io/joedmck/devcontainer-features
+- name: Features by CodeMan99
+  maintainer: Cody Taylor
+  contact: https://github.com/CodeMan99/features/issues
+  repository: https://github.com/CodeMan99/features
+  ociReference: ghcr.io/CodeMan99/features
```

```suggestion
  ociReference: ghcr.io/codeman99/features
```

Hi 👋

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.